frontend/src/stories/Toast.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+
+<Meta title="Components/Toast" />
+
+# Toast
+
+> Example: A success toast 
+
+Toasts are small generally one-line messages that appear to inform the user of the result of an action, or an update to the current view.
+
+## Structure and Options
+
+The toast consists of:
+
+- An optional icon
+- A text
+- A close button
+
+There are four types of toasts, each with a specific icon and colour associated with it:
```

Comment:
Do we need to specify the colours exactly here with HEX?

Reply:
Added named colours as discussed this morning.